server/src/universal/conf/delta-sharing-server.yaml.template

```diff
@@ -56,7 +56,7 @@ stalenessAcceptable: false
 # Whether to evaluate user provided `predicateHints`
 evaluatePredicateHints: false
 # Whether to evaluate user provided `jsonPredicateHints`
-evaluateJsonPredicateHints: false
+evaluateJsonPredicateHints: true
```

Comment:
To confirm this is intended.

Reply:
Yes, this is needed to enable the predicates.